examples/reference-entity-model/src/main/entity-config/databases/final-database.json

```diff
@@ -0,0 +1,9 @@
+{
+  "lang" : "zxx",
+  "path-namespace" : [ {
+    "prefix" : "es",
+    "namespace-uri" : "http://marklogic.com/entity-services"
+  } ],
+  "range-element-index" : [ ],
```

Comment:
Run "./gradle hubSaveIndexes" to generate the necessary indexes in these files.

Reply:
I ran "./gradlew hubSaveIndexes" that Rahul suggested above, and it output these files. I ran it again and there are no changes to the two generated files. "./gradle hubSaveIndexes" gives bash: ./gradle: is a directory in reference-entity-model.

Reply:
GIVEN properties having "facetable: true", WHEN "./gradlew hubSaveIndexes" is run THEN indexes are generated for properties marked facetable:true.

Are we not getting the indexes because "THEN" isnt implemented yet?

Reply:
@saarimrahman That's my mistake, I forgot the backend doesn't care about sortable yet.